### DISCUSSION TITLE
Increase Amount of Iterations and Steps above 250[FEATURE REQUEST]

### DISCUSSION THREAD

**Discussion by Darkarma** [hlky/stable-diffusion-webui#219] [Feature suggestions]
Requesting that there be a field next to the Iteration and Steps sliders to allow andtype a number above the 250 limit, please. This will be especially useful for long-duration runs.

**Comment by hlky** [comment]
You know steps above 250 has diminishing returns?

Would a command line toggle to force maximum batch size of the slider be good for you instead? just to keep the ui less cluttered, considering batch size > 250 is kinda advanced usage I assume you wouldn't mind and know how to use the cli arguments?

**Comment by Darkarma** [comment]
I'm still learning the system so I'd probably have to figure out what using cli arguments involves. But some sort of override would be nice, even if it's a command line interface.

Thank you. Also thank you for providing this great resource!

**Comment by hlky** [comment]
For reference 

[--help](https://github.com/hlky/stable-diffusion#--help)

[using cli arguments](https://github.com/hlky/stable-diffusion#want-to-run-full-precision)
